File: Lista2 java/src/Kolo.java
public class Kolo extends Figura{

	private final double r;

	public Kolo(double r) {
		this.r = r;
	}

	@Override
	public double pole() {
		return Math.PI * r * r;
	}

	@Override
	public double obwod() {
		return 2 * Math.PI * r;
	}

	public double getR() {
		return r;
	}

	@Override
	public String toString() {
		return "Kolo";
	}
}

File: lista2/src/Kolo.cpp
#include "funs.hpp"

Kolo::Kolo(double r) {
    this->r = r;
}

double Kolo::pole() {
    return 3.14 * r * r;
}

double Kolo::obwod() {
    return 2 * 3.14 * r;
}